cfgov/v1/models/sublanding_filterable_page.py

```diff
@@ -66,6 +66,11 @@ class SublandingFilterablePage(FilterableListMixin, CFGOVPage):
     ]
 
 
+class ResearchHubPage(CategoryFilterableMixin, SublandingFilterablePage):
+    template = "sublanding-page/index.html"
```

Comment:
```suggestion
```

Is this needed, since it's already inherited from the parent `SublandingFilterPage`?

Reply:
When I remove that line and try to load the page, I see `django.template.exceptions.TemplateDoesNotExist: v1/research_hub_page.html`

Reply:
Ah, you are right. Thanks for checking my assumption. Wagtail doesn't inherit the template from the parent but will instead try loading a template named for the specific page class.

Another way of avoiding the duplicate hardcoding might be to do:

```suggestion
    template = SublandingFilterablePage.template
```